Viper Practice/src/entity/Nutritions.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Nutritions.swift
+//  Viper Practice
+//
+//  Created by ahsan shabir on 02/04/2024.
+//
+
+import Foundation
+
+struct Nutritions: Codable {
+
+  var calories      : Float? = nil
+  var fat           : Float? = nil
+  var sugar         : Float? = nil
+  var carbohydrates : Float? = nil
+  var protein       : Float? = nil
+
+  enum CodingKeys: String, CodingKey {
```

Comment:
same comment for these coding keys as well

Reply:
coding keys removed